packages/babel-preset-gatsby/package.json

```diff
@@ -10,6 +10,9 @@
     "@babel/preset-react": "^7.0.0",
     "babel-plugin-macros": "^2.4.2"
   },
+  "peerDependencies": {
+    "@babel/core": "*"
```

Comment:
Should we be relatively safer here and pin to `^7.0.0`?

Reply:
Sure! Will update the PR - since the plugins are scoped to `^7.0.0` it should be fine 👍